src/pages/Products/components/MobileFilters/MobileFilters.js

```diff
@@ -0,0 +1,231 @@
+import React from "react";
+import { useFilter } from "../../../../context/FilterContext";
+import "./MobileFilters.css";
+
+const MobileFilters = ({ mobileFilters }) => {
+	const { state, dispatch } = useFilter();
+	const {
+		sliderRange,
+		priceRange,
+		paintings,
+		bags,
+		diary,
+		coaster,
+		rating,
+		sortBy,
+	} = state;
+
+	const resetHandler = () => {
+		dispatch({ type: "RESET", payload: {} });
+	};
+	return (
+		<div
+			className={
+				mobileFilters ? "mobile-filters" : `${"display-none mobile-filters"}`
+			}
```

Comment:
can you tell me why you are using mobileFilters ? "mobile-filters" : `${"display-none mobile-filters"}`?